<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>In2ConfigPrintUI</class>
 <widget class="QWidget" name="In2ConfigPrintUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>412</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>In2 Printing Solutions</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Impresi√≥n</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="m_cbNormal">
          <property name="text">
           <string>Normal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_cbReverse">
          <property name="text">
           <string>Reverse</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Color</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="m_cbMono">
          <property name="text">
           <string>Mono</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_cbGrayScale">
          <property name="text">
           <string>GrayScale</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>Offset</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Ptos</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_leOffsetPoints"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_leOffsetMm"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Velocidad</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="m_cbStandard">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Standard (&lt;50 m/min)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_cbHighSpeed">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>High Speed module</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Transporte y Disparo</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Modo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_cbModoImpresion">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>3</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>Trigered</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Fixed</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Back To Back</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>PD</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_cbTipoPD">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>3</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>External</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Internal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Absoluto</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_12">
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Alarmas</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="m_cbMonitorMacula">
            <property name="text">
             <string>Macula</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Maculas</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_leNMaculas">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="m_cbMonitorHeads">
          <property name="text">
           <string>Electronica</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="m_cbMonitorVision">
          <property name="text">
           <string>Vision</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_pbOK">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../In2Apps/In2Designer/In2Designer/shared.qrc">
           <normaloff>:/res/images/Ok_32x32.png</normaloff>:/res/images/Ok_32x32.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../In2Apps/In2Designer/In2Designer/shared.qrc"/>
 </resources>
 <connections/>
</ui>
